iroha/src/isi.rs

```diff
@@ -1,13 +1,15 @@
-use crate::{
-    account::isi::CreateAccount,
-    asset::isi::{AddAssetQuantity, TransferAsset},
-    domain::isi::CreateDomain,
-    prelude::*,
-    wsv::WorldStateView,
-};
+use crate::{account, asset, domain, peer, wsv::WorldStateView};
 use parity_scale_codec::{Decode, Encode};
 
 /// Identification of an Iroha's entites. Consists of Entity's name and Domain's name.
+///
+/// # Example
+///
+/// ```
+/// use iroha::isi::Id;
+///
+/// let id = Id::new("gold", "mine");
+/// ```
 #[derive(Clone, Debug, PartialEq, Eq, std::hash::Hash, Encode, Decode)]
 pub struct Id(pub String, pub String);
```

Comment:
Maybe implement `Copy` for `Id` in this RR many lines have `account_id.clone()`, so why not. It is not expensive to clone it, therefore we shouldn't worry over implicit clone.

Reply:
1. String is not marked as Copy: `A simple bitwise copy of String values would merely copy the pointer, leading to a double free down the line.`
1. I'm not sure that cloning of two string will not be expensive in our case and I hope we will remove all `clone` calls soon.

I will insist to do not derive `Copy` in cases where our struct contains more than integers or fixed size arrays.

Reply:
I meant implementing it so that we clone the string. But okay, I guess it's a reasonable policy to copy only, where we can derive.